src/python_be.cc

```diff
@@ -661,46 +660,33 @@ ModelInstanceState::ExecuteBLSRequest(
             lbackend_memory.reset(backend_memory);
             input_tensor->SetMemory(std::move(PbMemory::Create(
                 Stub()->ShmPool(), std::move(lbackend_memory))));
+            gpu_buffer_transporter.AddBuffer(
```

Comment:
instead of `transporter` would `response` work? `GPUBuffers` and `GPUBuffersResponse` instead of `GPUBuffersShm` and `GPUBufferTransporter`

Reply:
I like transporter more as I think "response" may imply that it can only be used with the backend responses which is not true. Let me know if you have any other suggestions.